miniapp/include/dlaf/miniapp/options.h

```diff
@@ -169,7 +169,6 @@ T stringToBlasEnum(const std::string& option_name, const std::string& x,
         valid_values_stream << ", ";
       }
     }
-    std::string option_name_dashes = "--" + option_name;
     DLAF_MINIAPP_INVALID_OPTION_VALUE(option_name, x, valid_values_stream.str());
```

Comment:
I think this should rather be:
```suggestion
    DLAF_MINIAPP_INVALID_OPTION_VALUE(option_name_dashes, x, valid_values_stream.str());
```